src/main/resources/edu/cornell/kfs/fp/businessobject/datadictionary/CorporateBilledCorporatePaidTransactionDetail.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+	<bean id="CorporateBilledCorporatePaidTransactionDetail" parent="CorporateBilledCorporatePaidTransactionDetail-parentBean" />
+	<bean id="CorporateBilledCorporatePaidTransactionDetail-parentBean" abstract="true" parent="ProcurementCardTransactionDetail">
+		<property name="businessObjectClass" value="edu.cornell.kfs.fp.businessobject.CorporateBilledCorporatePaidTransactionDetail" />
+		<property name="objectLabel" value="Corporate Billed Corporate Paid Transaction Detail" />
+		<property name="inquiryDefinition">
+			<ref bean="CorporateBilledCoporatePaidTransactionDetail-inquiryDefinition" />
+		</property>
+	</bean>
+
+	<bean id="CorporateBilledCoporatePaidTransactionDetail-inquiryDefinition" parent="ProcurementCardTransactionDetail-inquiryDefinition-parentBean">
```

Comment:
Should the CBCP inquiry definition extend from the PCDO one, instead of the PCDO "parentBean" one?